### PR TITLE
Add Jenkinsfile for build automation of docker image

### PR DESCRIPTION
**Pull request by jamesrgregg**
- Add Jenkinsfile
- Pushes images to Nexus
- Kicks off Clair scan of the images

Signed-off-by: James Gregg <james.r.gregg@intel.com>

### CODE REVIEW

**Comment by jamesrgregg** [comment]
@ernestojeda @lranjbar please review

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    }
+                }
+            }
+        }
+
+        stage('Clair Image Scan') {
+            when { expression { edgex.isReleaseStream() } }
+            steps {
+                edgeXClair("${env.DOCKER_REGISTRY}:10004/docker-edgex-consul:1.0.0")
+                edgeXClair("${env.DOCKER_REGISTRY}:10004/docker-edgex-consul-arm64:1.0.0")
```
Comment (ernestojeda) [inline Jenkinsfile]: Wrong tag, should be `1.1.0`

**Review by ernestojeda — APPROVED** [review]
Version needs updating

**Review by ernestojeda — CHANGES_REQUESTED** [review]
See my comment about the Clair stage

File: Jenkinsfile (hunk 1 above)
Comment (jamesrgregg) [reply Jenkinsfile]: oh snap!!

**Review by ernestojeda — APPROVED** [review]
LGTM 👍

**Pull request merged**